tests/backends/test_redis.py

```diff
@@ -3,7 +3,7 @@
 from pytest_asyncio.plugin import SubRequest
 
 from starsessions.stores.base import SessionStore
-from starsessions.stores.redis import RedisStore
```

Comment:
This should not conflict with top-level dependency. What was the issue?
Renaming file may be backward incompatible.

Reply:
yep, not sure why I have thought that when I switched locally, I could not import stuff, but it's ok without a rename indeed